src/components/common/Certify.vue

```diff
@@ -73,6 +74,13 @@ export default class Certify extends Vue {
   /** Entity Display prop. */
   @Prop({ default: '' }) readonly entityDisplay!: string
 
+  /** Prompt the validations. Used for global validations. */
+  @Prop({ default: false }) readonly validateForm!: boolean
+
+  certifyName = ''
+
+  checkboxState = false;
```

Comment:
Did you run `npm run lint`? Did it complain about the trailing semi-colon?

Reply:
I did, it didn't make any changes there but I can remove it

Reply:
Yes, please. Thanks!

Reply:
Did it! Should be in the new commit